admin_guide/high_availability.adoc

```diff
@@ -75,66 +188,119 @@ The list of command options described here are a subset that are relevant to thi
-
-****
-`$ osadm ipfailover ha-geo-cache --replicas=4 --selector="ha-cache=geo" --virtual-ips=10.245.2.101-104 --watch-port=9736 --credentials="$OPENSHIFTCONFIG" --create`
-****
-
-==== Internals
-The osadm ipfailover command ensures that an ipfailover sidecar pod runs on each of the nodes matching the constraints or label we used in step 1. This ipfailover sidecar pod uses VRRP (Virtual Router Reduncy Protocol) within keepalived to monitor that the service on the watched port is available and keepalived will automatically float the VIPs in event of the service not being available.
-
-In our example, we can now use the VIPs 10.245.2.101 through 10.245.2.104 to send traffic to the geo-cache service. If a particular geo-cache instance is "unreachable" (as an example due to a node failure), keepalived will float the VIPs automatically float amongst group of nodes we labelled "ha-cache=geo" and the service would still be reachable via those virtual IP addresses.
-
-== Examples
-
-See what the IP failover configuration would look like if it is created:
-
-****
-`$ osadm ipfailover <options ...> -o json`
-****
-
-
-Create an IP failover configuration if it does not already exist:
-
-****
-`$ osadm ipfailover --virtual-ips="1.2.3.4-5,6.7.8.9,10.11.12.13-15" --create`
-****
-
-
-Create an IP failover configuration on a selection of nodes labelled "my-ha-service=har-reporter" (on 4 nodes with 7 virtual IPs monitoring a service listening on port 4242.
-
-****
-`$ osadm ipfailover harreporter --selector="my-ha-service=har-reporter" --virtual-ips="10.245.2.42,10.245.2.100-104,10.245.2.142,10.245.2.242" --watch-port=4242 --replicas=7 --create`
-****
+////
+
+The `osadm ipfailover` command ensures that a pod with failover runs on each of
+the nodes matching the constraints or label used. This pod uses a VRRP (Virtual
+Router Redundancy Protocol) with link:http://www.keepalived.org/[Keepalived] to ensure that the service on the
+watched port is available, and, if needed, Keepalived will automatically float
+the VIPs if the service is not available.
+
+=== Configuring a Highly-available Routing Service
+The following steps describe how to setup a highly-available router environment with IP failover:
+
+. Label the nodes for the service. This step can be optional if you run the
+service on any of the nodes in your Kubernetes cluster and use VIPs that can
+float within those nodes. This process may already exist within a complex
+cluster, in that nodes may be filtered by any constraints or requirements
+specified (e.g. nodes with SSD drives, or higher cpu/memory/disk requirements,
+etc.).
++
+The following example defines a label as router instances that are servicing
+traffic in the US west geography "ha-router=geo-us-west":
++
+====
+----
+$ osc label nodes openshift-minion-{5,6,7,8,9} "ha-router=geo-us-west"
```

Comment:
Can we change "minion" to "node" in the example node name here? Just for consistency (see https://github.com/openshift/openshift-docs/pull/412).